Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the offical golang image to create a binary.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.20.2-buster as builder
```

Comment:
Note:
 Requirement was to use golang:alpine as builder.

Reply:
Fixed